make/Makefile.container.mk

```diff
@@ -60,5 +60,10 @@ else
 	podman push ${QUAY_TAG}
 endif
 
+## container-multi-arch-push-kiali-quay: Pushes the Kiali multi-arch image to quay.
+container-multi-arch-push-kiali-quay: .prepare-kiali-image-files
+	@echo Pushing Kiali multi-arch image to ${QUAY_TAG} using docker buildx
+	docker buildx build --push $(foreach arch,${TARGET_ARCHS},--platform=linux/${arch}) $(foreach tag,${QUAY_TAG},--tag=${tag}) -f ${OUTDIR}/docker/Dockerfile-multi-arch ${OUTDIR}/docker
+
```

Comment:
could pass QUAY_TAG="quay.io/kiali/kiali:1.26 quay.io/kiali/kiali:1.26.1" in jenkins for multi tag publish